sandbox/component/button/index.njk

```diff
@@ -0,0 +1,409 @@
+{% extends '_layout.njk' %}
+{% set title = 'Button' %}
+
+{% import 'cwel/component/button/button.html' as button %}
+
+{% block content %}
+
+     {% set config = {
```

Comment:
@NathanRodger Nothing wrong with this now. Would be nice to create an issue as a reminder to grab this from a json file. Need to write the necessary nunjucks function/filter to convert strings into objects, though.